### DISCUSSION TITLE
Setting the z home for a new (different) bed

### DISCUSSION THREAD

**Discussion by jorythompson** [luc-github/Repetier-Firmware-0.92#385] [Q&A]
My glass bed cracked on my davinci/repetier and rather than just lay a new plate of glass over the broken bed, I did this:
http://www.soliforum.com/topic/17680/davinci-10a-bed-replacment/
Unfortunately, there is no mention of how to handle the thermistor so I just taped the original one (the one that came with the new heated bed had vastly different resistance) to the bottom of the heated bed with kapton tape (seems to be working).
Now my problem is the z home. I have leveled the bed and updated the eeprom but when I try to print with Repetier Host, the bed goes up too high and the filament is smeared on to the bed instead of a nice bead. 
Any suggestions?

**Comment by luc-github** [comment]
you must level the bed with the screw first then adjust the z offset in EEPROM, did you do manual leveling with sheet of paper and the 3 screws under the bed ?

**Reply by jorythompson** [reply]
First, I moved the bed down with the z offset (remember that this is a new, different bed that is slightly higher than the original.) 

Then, I leveled everything with the screw and used paper to make sure that head is the correct height.

When I home the axis's, the bed is at the correct height for a piece of paper to slip between the head and the bed. 

Then I saved the settings in the EEPROM. 

When I print, the head is too close to the bed

**Reply by luc-github** [reply]
when doing manual bed leveling zoffset must be 0 in EEPROM, then you do manual leveling, not the oposite, at home position z position should be 0,
is that the case ?
manual bed leveling must be done with bed and extruder hot

**Comment by jorythompson** [comment]
If I set z to zero and press home, the bed is too high even when I screw in
the adjustment screws.

**Reply by luc-github** [reply]
then you need to move the flag of the end stop https://www.google.com/search?q=thingiverse+davinci+end+stop
